views/index.pug

```diff
@@ -0,0 +1,84 @@
+html
+head
+    link(rel="stylesheet" href="/style.css")
+    script(src="../bagdata.js")
+    body
+     h1.title.headersCentered Faith's Bag Collection 
+     h2.headersCentered What bags are you interested in ?
+    ul.list 
+      p Choose one of the following:
+      li.listItems 
+        p 
+          strong GET 
+          | /allBags
+        div#getAll
+            p Method: GET 
+            p Route: http://localhost:2035/bags/allBags 
```

Comment:
I would expect this route to simply be http://localhost2035/bags, the allBags is redundant